lib/turn.rb

```diff
@@ -0,0 +1,68 @@
+class Turn
+    attr_reader :player1, :player2, :spoils_of_war
+    def initialize(player1, player2)
+        @player1 = player1
+        @player2 = player2
+        @spoils_of_war = []
+        @turn_type = nil
```

Comment:
Saving the turn type into an instance variable is a very smart move and that way you only have to call the `type` method once per turn. Great idea!